### PR TITLE
Inilne style z-index magic number to avoid perseus overlay

### PR DESCRIPTION
**Pull request by nucleogenesis**
### Summary

Fixes issue noted here: 

https://github.com/learningequality/kolibri/issues/6327

### Reviewer guidance

Go to create a quiz, make sure that your page showing the quiz questions is sized so that it would go off the bottom of the screen.

The perseus renderer content will not overlay the bottom bar.

This issue isn't found in the quiz taking part as a learner, but only in quiz creation.

So I just added the style inline rather than making the whole bottom bar component have a wacky z-index.

### References

https://github.com/learningequality/kolibri/issues/6327

----

### Contributor Checklist

PR process:

- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [x] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

Testing:

- [x] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [ ] Critical and brittle code paths are covered by unit tests

### Reviewer Checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

**Comment by jonboiser** [comment]
Maybe to make the code a little nicer, could put this rule in a CSS class placed on the BottomAppBar and defined inside of CreateExamPreview? Also, are we sure perseus widgets don't have z-indices higher than 1061?

**Review by jonboiser — APPROVED** [review]

**Comment by rtibbles** [comment]
Builds stalled on RPi step.

File: kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/CreateExamPreview.vue [hunk 1]
```diff
@@ -105,7 +105,7 @@
         :selectedExercises="selectedExercises"
       />
 
-      <BottomAppBar>
+      <BottomAppBar style="z-index: 1062;">
```
Comment (indirectlylit) [inline kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/CreateExamPreview.vue:108]: where does this number magic come from?

This is fine for now, but generally, our z-indexes should map to Material elevations - https://material.io/design/environment/elevation.html#default-elevations

**Comment by rtibbles** [comment]
I think we will want to do some materialisation of perseus z indices to properly handle this

**Comment by rtibbles** [comment]
Alternatively, we can just chuck perseus inside an iframe and stop it leaking.

**Pull request merged**